projects/1.20/assets/forestry-community-edition/forestry/lang/zh_cn.json

```
@@ -0,0 +1,2360 @@
+{
+  "_comment": "###################",
+  "_comment": "CORE",
+  "_comment": "###################",
+  "death.engine.clockwork.player": "由于%2$s的粗心大意，%1$s被散落的弹簧片刺中了。",
+  "death.engine.clockwork": "%1$s被散落的弹簧片刺中了。",
+  "for.chat.cannotmemorizeagain": "你已经记住了相关内容。",
+  "for.chat.memorizednote": "你记录了笔记并扔掉了皱巴巴的小纸片。",
+  "for.chat.memorizednote2": "你对用%1$s和%2$s杂交出%3$s更有信心了！",
+  "for.chat.command.aliases": "命令代称：%s",
+  "for.chat.command.list": "可用子命令：",
+  "for.chat.command.permlevel": "权限等级：%s",
+  "for.chat.command.noperms": "你没有权限使用该命令。",
+  "for.chat.command.forestry.help": "林业模组可用的许多命令",
+  "for.chat.command.forestry.format": "格式：/%s <子命令>",
+  "for.chat.command.forestry.version.desc": "- %s：版本信息",
+  "for.chat.command.forestry.version.help": "显示当前版本",
+  "for.chat.command.forestry.version.format": "格式：/%s",
+  "for.chat.command.forestry.modules.desc": "- %s：林业组件列表",
+  "for.chat.command.forestry.modules.help": "列出当前的林业组件。绿色为启用的，红色为禁用的。",
+  "for.chat.command.forestry.modules.format": "格式：/%s",
+  "for.chat.command.forestry.modules.info.desc": "- %s：林业组件信息",
+  "for.chat.command.forestry.modules.info.help": "显示所给林业组件的信息",
+  "for.chat.command.forestry.modules.info.format": "格式：/%s <组件名>",
+  "for.chat.help": "输入“%s”查看帮助。",
+  "for.chat.modules.error": "组件%s无可查看信息。",
+  "for.chat.version": "适用于Minecraft %2$s的林业%1$s（最新版本：%3$s）。",
+  "for.circuit.dampener.1": "阻尼",
+  "for.circuit.dampener.1.description.1": "最大输出减少到10RF/t",
+  "for.circuit.dampener.1.description.2": "保留燃料输出",
+  "for.circuit.electric.boost.1": "升压 I",
+  "for.circuit.electric.boost.1.description.1": "输出增加10RF/t",

```

```suggestion
  "for.circuit.electric.boost.1.description.1": "输出功率增加10RF/t",

```
不确定是否有必要写为“输出/输出功率增加”之类更加精确的描述？

---

同上，也不存在了，本来应当是给EU转RF的引擎用的
改不改都随意吧，改一下也没问题